appsembleredx/monkeypatch.py

```diff
@@ -24,38 +24,38 @@ class fakemodule(object):
 
         import sys
         logger.warn("Setting fake certificates.views.support module for CMS.  Not used in Studio")
-        sys.modules['certificates.views.support'] = fakemodule()  # load an emtpty module
+        sys.modules['certificates.views.support'] = fakemodule()  # noqa: load an emtpty module
```

Comment:
hmm... yes, good catch @thraxil it looks like the indentation got messed up somehow.  `fakemodule()` should not be inside of the class definition